enocean/eep_parser.py

```diff
@@ -13,11 +13,14 @@ def CanParse(self, eep):
         return found
 
     def Parse(self, eep, payload, status):
-        #self.logger.debug('enocean: parser called with eep={} / payload={} / status={}'.format(eep, payload, status))
+        self.logger.debug('enocean: parser called with eep = {} / payload = {} / status = {}'.format(eep, ', '.join(hex(x) for x in payload), hex(status)))
```

Comment:
This line is useful for local debugging and development. But please comment out for pull request.